js/components/dateRange.js

```diff
@@ -152,9 +138,10 @@ Fliplet.FormBuilder.field('dateRange', {
       this.$emit('_input', this.name, val, false, true);
     },
     selectedRange: function(range) {
+      var $vm = this;
```

Comment:
Why doesn't this work?

```suggestion
      this.value = this.getDate(range.value)
```